src/libraries/System.Formats.Asn1/src/System.Formats.Asn1.csproj

```diff
@@ -46,10 +47,26 @@
     <Compile Include="System\Formats\Asn1\TagClass.cs" />
     <Compile Include="System\Formats\Asn1\UniversalTagNumber.cs" />
   </ItemGroup>
-  <ItemGroup>
-    <PackageReference Include="System.Buffers" Version="$(SystemBuffersVersion)" />
-    <PackageReference Include="System.Memory" Version="$(SystemMemoryVersion)" />
-  </ItemGroup>
+  <Choose>
```

Comment:
We usually don't use `Choose/When` statements in csprojs but I'm fine with it, it is simpler.